test/diameter_Gy_SUITE.erl

```diff
@@ -94,6 +94,8 @@
 	       },
 	  apps =>
 	      #{default =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init => [#{service => <<"Default">>}],
 		      authenticate => [],
 		      authorize => [],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok